Week3/week3-homework/todo-app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const { showTable, addTodoItem, deleteTodoItem, addTodoList, deleteTodoList, markCompleted, addReminder } = require('./actions/action')
+const bodyParser = require('body-parser');
+const app = express();
+app.use(bodyParser.json());
+
+
+app.get('/:table', (req, res) => {
+  showTable(req, res)
+});
+
+app.post('/add_todo_item', (req, res) => {
```

Comment:
This path does not conform to REST. The path should be simply `/todo_item`, the method type `post `already communicated the intention to create an object. Same for delete it should be simply
`app.delete('/todo_item/:id' ...`
The method is `delete `so there is no need to have delete in the path as well.